tracer/controllers/default.py

```diff
@@ -136,6 +137,15 @@ def status_code(self):
 			code = 104
 		return code
 
+	def create_reboot_required_file(self):
+		"""
+		If a reboot is needed, create a /var/run/reboot-required file.
```

Comment:
The correct location is `/run`, not `/var/run`. In most cases, this doesn't matter greatly, because a compat symlink is provided, but when `/var` is a separate partition, `/var/run` would only be accessible if `/var` is mounted, i.e. rather late in boot. `/run` is always available. `/var/run` is a compat symlink that should not be used (esp. in new code).

Reply:
Thank you @keszybz,
if I change the location, will this be okay with you @major?

Reply:
@FrostyX should be fine!